app/views/blocks/profile/_socials.html.erb

```diff
@@ -1,14 +1,14 @@
 <div class="content-block">
   <% if block_property(block, "battlenet").present? %>
     <div class="content-block__social">
-      <%= inline_svg_tag "icons/icon-battlenet.svg", height: 21, width: 21 %>
+      <%= inline_svg_tag "icons/icon-battlenet.svg", height: 21, width: 21, class: "icon-battlenet" %>
```

Comment:
This would go with the review above
```suggestion
      <%= inline_svg_tag "icons/icon-battlenet.svg", height: 21, width: 21, class: "text-battlenet" %>
```